### PR TITLE
Stop sending email when somebody completes the hoc census form

### PR DESCRIPTION
**Pull request by drewsamnick**
We are currently sending an email when somebody signs up for HOC and then a second email after they complete the census form. We don't want to send two emails so this will disable the census one.

### CODE REVIEW

File: pegasus/forms/hoc_census.rb [hunk 1]
```diff
@@ -28,6 +28,6 @@ def self.normalize(data)
   end
 
   def self.receipt
```
Comment (Erin007) [inline pegasus/forms/hoc_census.rb:30]: Do we even this self.receipt method?

Comment (drewsamnick) [reply pegasus/forms/hoc_census.rb:30]: We need to override the method on the parent (HocSignup2017)

Comment (Erin007) [reply pegasus/forms/hoc_census.rb:30]: Got it.

**Comment by drewsamnick** [comment]
[Turn off email going to people who fill out the census on the HOC site](https://trello.com/c/d5x4mzWo/34-turn-off-email-going-to-people-who-fill-out-the-census-on-the-hoc-site)

**Pull request merged**